src/shared/components/ProfilePage/Stats/ChartTooltip/index.jsx

```diff
@@ -19,6 +20,10 @@ const ChartTooltip = ({
       pointerEvents: href ? 'all' : 'none',
     }}
     href={href}
+    onClick={(e) => {
```

Comment:
Why using `onClick()` to redirect URL instead update the `href` params.

This will give some issues like if right click and select "open in new tab", `onClick()` will be ignore.